cmd/send.go

```diff
@@ -10,6 +10,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
+func checkBalanceBeforeSend(tokens, fee float64) {
+	wg := &sync.WaitGroup{}
+	statusBar := &ZCNStatus{wg: wg}
+	wg.Add(1)
+	err := zcncore.GetBalance(statusBar)
+	if err != nil {
+		return // continue sending txn even if getBalance fails
```

Comment:
what does "fails" mean?
A balance of zero will return a 400 right now, this is a client side "Failure"
(This is not ideal and a balance of zero should be returned with a 200 instead)

Reply:
Yes fail is meant to suggest any response other than 200, the idea being that if for some reason getBalance fails, it should not also stop the user from being able to do the the transaction.